### ISSUE TITLE
react-native run-ios

### ISSUE THREAD

**Issue by alokt** [GeekyAnts/ignite-native-base-boilerplate#12]
After following steps i'm getting below error. Any advice?

Command /bin/sh failed with exit code 1

** BUILD FAILED **

The following commands produced analyzer issues:
	Analyze /Users/at/dev/React-Native/BoilerTest/node_modules/react-native/ReactCommon/yoga/yoga/YGNodeList.c
	Analyze /Users/at/dev/React-Native/BoilerTest/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.c
(2 commands with analyzer issues)

The following build commands failed:
	PhaseScriptExecution Install\ Third\ Party /Users/at/dev/React-Native/BoilerTest/ios/build/Build/Intermediates.noindex/React.build/Debug-iphonesimulator/double-conversion.build/Script-190EE32F1E6A43DE00A8543A.sh
(1 failure)

Installing build/Build/Products/Debug-iphonesimulator/BoilerTest.app
An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=2):
Failed to install the requested application
An application bundle was not found at the provided path.
Provide a valid path to the desired application bundle.
Print: Entry, ":CFBundleIdentifier", Does Not Exist

Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/Build/Products/Debug-iphonesimulator/BoilerTest.app/Info.plist
Print: Entry, ":CFBundleIdentifier", Does Not Exist

**Comment by BillyFigueroa** [comment]
I m having same issue

**Comment by jraleman** [comment]
I have encountered this problem before. In layman's term, it can't find the bundle file.
To generate one, run the following snippet, replacing `{project-name}` with yours:

```
node node_modules/react-native/local-cli/cli.js \
bundle --entry-file='index.ios.js' \
       --bundle-output='./ios/{ project_name }/main.jsbundle' \
       --platform='ios' \
       --assets-dest='./ios' \
       --dev=false
```

Hope this helps!
